database/migrations/2022_06_18_113401_tasks_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tasks', function (Blueprint $table) {
+            $table->string('date');
+            $table->string('time');
+        });
```

Comment:
Does this work? you are trying to create the table again when you are supposed to add two new fields.

Unique constraint is also missing